### ISSUE TITLE
Update Project Profile: Access the Data (Change Status to On Hold, Remove AJ Price & Jason Yung)

### ISSUE THREAD

**Issue by JessicaLucindaCheng** [hackforla/website#6064]
### Prerequisites
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started) and attend an onboarding session.
2. You have already read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md).

### Overview
We need to keep project information up to date so that visitors to the website can find accurate information.

### Action Items
- [x] In your IDE, open the `_projects/access-the-data.md` file
- [x] Observe the existing syntax of the front matter block [^1] in the file
- [x] Find the `leadership` variable and remove the following:
```
  - name: AJ Price
    role: Product Manager
    links:
      slack: 'https://hackforla.slack.com/team/U02UZ5BK8UB'
      github: 'https://github.com/mxajPrice'
    picture: https://avatars.githubusercontent.com/mxajPrice
  - name: Jason Yung
    role: Project Manager
    links:
      slack: 'https://hackforla.slack.com/team/U059W9P96C8'
      github: 'https://github.com/merlinsmagic'
    picture: https://avatars.githubusercontent.com/merlinsmagic
```
- [x] Find the `status` variable and change the status
**From**
```
status: Active
```
**To**
```
status: On Hold
```
- [x] Verify the changes by viewing the following in your local environment and include before and after screenshots with your pull request:
  - [x] Access the Data page [^2]
  - [x] the project's card on the Projects' page [^3]
- [x] Once your pull request is merged, go to #6018 and
  - [x] Check off this issue under the _Dependency_ section
  - [x] If all the issues in the _Dependency_ section are checked off, move #6018 to **Questions / In Review** column and uncheck the **Dependency label**.
    - <details>
        <summary>Click here to see how to uncheck the <b>Dependency label</b></summary>
        <img src="https://github.com/hackforla/website/assets/31293603/6f53f4d4-7d2c-45f8-8534-9936fc9adee8" width="300px">
      </details>

### Resources/Instructions
- This issue was initiated by #6018
[^1]: [Info about the front matter block](https://jekyllrb.com/docs/front-matter/)
[^2]: Project detailed info page URL: https://www.hackforla.org/projects/access-the-data
[^3]: Projects page URL: https://www.hackforla.org/projects/

**Comment by iancooperman** [comment]
Availability: 1/20 10am-1pm, 8pm-9pm; 1/21 3pm-9pm; 1/22 5pm-9pm
ETA: Monday, 1/22/24
